### ISSUE TITLE
Incorrect path on Mac

### ISSUE THREAD

**Issue by shadowmint** [Rabadash8820/UnityAssemblies#5]
The default path on mac is set to `/Application/Unity/Hub/Editor/...` however, this is wrong.

The path should be `/Applications/Unity/Hub/Editor/...` note the 's'. /Application is not a folder:

```
$ ls /Application | grep Unity
ls: /Application: No such file or directory

$ ls /Applications | grep Unity
Unity Hub.app
```

This appears to be from here https://github.com/DerploidEntertainment/UnityAssemblies/blob/c81c384f51837d27b811d4b80b2dd824552ae575/nupkg/build/Unity3D.props#L6 

Unfortunately adding: 

```
<UnityInstallRoot>/Applications/Unity/Hub/Editor</UnityInstallRoot>
```

Doesn't fix it because then you get:

```
warning MSB3106: Assembly strong name "warning MSB3106: Assembly strong name "/Applications/Unity/Hub/Editor/2019.4.15f1/Editor/Data/Managed/UnityEngine.dll" is either a path which could not be found or it is a full assembly name which is badly formed.
```

Where the correct path is actually:

```
ls /Applications/Unity/Hub/Editor/2019.4.15f1/Unity.app/Contents/Managed/
ExCSS.Unity.dll
Unity.Cecil.dll
....
```

Anyway, long story short: sadly, this doesn't really work on macs.

**Comment by Rabadash8820** [comment]
@shadowmint Thanks for reporting this. Sorry for the delayed response, I just realized that I hadn't set GitHub to watch all activity in this repo! Anyway, you are correct, the root MacOS paths are wrong. I work on a Windows machine, and I never actually tested the paths out on a Mac when I copied them from Unity's docs. So yeah, I'll take the blame for some egregious false advertising there. 😬 Luckily, somebody else made a fork of this repo and corrected the paths, so I was able to copy those back over into this repo (see the `mac-and-2020.2` branch). I plan to have the NuGet package updated within a day or two.

**Comment by Rabadash8820** [comment]
Okay @shadowmint I just published [v1.6.0](https://www.nuget.org/packages/Unity3D/#) on NuGet. If you use a Mac, could you please verify that the default paths are working correctly in that version?

**Comment by hoonsubin** [comment]
@Rabadash8820 Not the OP, but I can confirm that the path is correct from my build

**Comment by Rabadash8820** [comment]
@hoonsubin Works for me, thanks for checking that!
